proposals/repeatable-annotations.md

```diff
@@ -130,19 +134,19 @@ annotation class Tag(val name: String) {
     // @Target(CLASS, FUNCTION)
     // @Retention(BINARY)
     // @kotlin.jvm.internal.RepeatableContainer
-    // /* synthetic */ annotation class Container(val value: Array<Tag>)
+    // public annotation class Container(val value: Array<Tag>)
 }
 ```
 
-Another error is introduced in case the container annotation is applied manually at the same time as the contained annotation:
+Another error is introduced in case the container annotation is applied manually at the same time as the contained annotation, if the latter is repeated:
```

Comment:
But is it legal in Java to have both a container and a single copy of annotation? Is it available via Java reflection's `getAnnotationsByType`? If we support it just for the rare case of backwards compatibility, then we should have a migration plan to get rid of it.

Reply:
Yes, apparently it is legal in Java, and `getAnnotationsByType` returns both annotations. (The previous text about `getAnnotationsByType` in the proposal was incorrect.)
```
@Tag("1")
@Tags(@Tag("2"))
public class A {
    public static void main(String[] args) {
        System.out.println(Arrays.asList(A.class.getAnnotationsByType(Tag.class)));
    }
}
```
->
```
[@test.Tag(value="1"), @test.Tag(value="2")]
```

With this in mind, I don't see much value in prohibiting this case. It's not especially confusing either, just a bit weird...

Reply:
Ok.